pkg/network/usm/sharedlibraries/ebpf.go

```diff
@@ -150,7 +152,7 @@ func (e *EbpfProgram) setupManagerAndPerfHandlers() {
 		}
 		mgr.PerfMaps = append(mgr.PerfMaps, pm)
 		ebpftelemetry.ReportPerfMapTelemetry(pm)
-		e.getLibsetData(libset).perfHandler = perfHandler
+		e.libsets[libset].perfHandler = perfHandler
```

Comment:
Having a small concern about an edge case, and I'm not sure if it is feasible
but we might have (in the future as of mistake, or a code that we've missed) a libset in `LibsetToLibSuffixes` that does not exists in `e.Libsets`. In such a case we will panic.

Why can't we iterate over `e.libsets` and avoid this edge case?

Reply:
Changed, it simplifies the code too so it looks better.